### DISCUSSION TITLE
Different color on connection

### DISCUSSION THREAD

**Discussion by MagnusOren** [mattieserver/netbox-topology-views#465] [Q&A] [answered]
Hello,

I was wondering if it would be possible to differentiate the colors of cables in our topology based on their status. Currently, all cables are blue, but I would like connected cables to be represented in green.

**Comment by dreng** [comment, marked as answer]
Hi Magnus,

You can edit cables in NetBox and give them different colors. The topology plugin uses the defined colors. If all of your cables haven't got any color definied, the topology plugin will represent them all in blue.

The idea is to enter the actual colour of the cable in NetBox. But if you prefer to select the colours by state, that is your decision.
